Resources/views/Collector/migrations.html.twig

```diff
@@ -21,25 +21,35 @@
             {% endset %}
 
             {% set text %}
-                <div class="sf-toolbar-info-piece">
-                    <b>Current Migration</b>
-                    <span>{{ executed_migrations > 0 ? collector.data.executed_migrations|last.version|split('\\')|last : 'n/a' }}</span>
+                <div class="sf-toolbar-info-group">
+                    <div class="sf-toolbar-info-piece">
+                        <b>Current Migration</b>
+                        <span>{{ executed_migrations > 0 ? collector.data.executed_migrations|last.version|split('\\')|last : 'n/a' }}</span>
+                    </div>
                 </div>
-                <div class="sf-toolbar-info-piece">
-                    <b>Executed Migrations</b>
-                    <span class="sf-toolbar-status">{{ executed_migrations }}</span>
-                </div>
-                <div class="sf-toolbar-info-piece">
-                    <b>└─ Unavailable</b>
-                    <span class="sf-toolbar-status {{ unavailable_migrations > 0 ? 'sf-toolbar-status-yellow' }}">{{ unavailable_migrations }}</span>
-                </div>
-                <div class="sf-toolbar-info-piece">
-                    <b>└─ Available</b>
-                    <span class="sf-toolbar-status">{{ available_migrations }}</span>
-                </div>
-                <div class="sf-toolbar-info-piece">
-                    <b>└─ New</b>
-                    <span class="sf-toolbar-status {{ new_migrations > 0 ? 'sf-toolbar-status-red' }}">{{ new_migrations }}</span>
+
+                <div class="sf-toolbar-info-group">
```

Comment:
As you updated the toolbar unconditionally, please also post screenshots of how it looks on 6.1 now

Reply:
indeed